docs/integrations/sources/google-analytics-v4.md

```diff
@@ -96,6 +102,7 @@ added by default to all reports. There are 8 default reports. To add more report
 
 | Version | Date       | Pull Request                                             | Subject                                            |
 |:--------|:-----------|:---------------------------------------------------------|:---------------------------------------------------|
+| 0.1.2   | 2023-02-06 | [22416](https://github.com/airbytehq/airbyte/pull/22416) | Improved section on service account                   |
```

Comment:
Because it is only a documentation update you don't need to bump the version. Please remove this line.